src/main/java/Main.java

```diff
@@ -1,5 +1,20 @@
+import java.io.IOException;
+
 class Main {
     public static void main(String[] args) {
-        // uzupełnij rozwiązanie
+        String fileName1 = "operations.txt";
+        String fileName2 = "operationsResults.txt";
+        MathOperationsWriter operationsWriter = new MathOperationsWriter();
+        MathOperationsReader operationsReader = new MathOperationsReader();
+        Calculator calculator = new Calculator();
+        try {
+            operationsWriter.writeToFile("1 + 2 \n" + "2 + 2 \n" + "3 - 1", fileName1);
```

Comment:
Z powodu tego wiersza testy nie działają poprawnie, bo zawsze nadpisujesz plik swoją wartością.
Jeżeli zakomentujesz ten wiersz, to zobaczysz, że rozwiązanie daje błędne wyniki i nie działa z wartościami typu double